File: JavaPatternPrograms/SymbolPatterns/Pattern 8.java
class Pattern_54
 {
     public static void main(String args[])
     {
 int size=3;
  
 
 for(int i=size;i>=-size;i--){
 for(int j=size;j>=Math.abs(i);j--)
 {
 System.out.print("*");
 }
 System.out.println();
 }
     } //end of main
 }  //end of class

File: CSharpPatternPrograms/SymbolPatterns/Pattern 8.cs
using System;

class Pattern_54
{
    public static void Main(string[] args)
    {
        int size = 3;


        for (int i = size; i >= -size; i--)
        {
            for (int j = size; j >= Math.Abs(i); j--)
            {
                Console.Write("*");
            }
            Console.WriteLine();
        }
        Console.ReadKey(true);
    } //end of main
} //end of class
